.pre-commit-config.yaml

```diff
@@ -1,4 +1,4 @@
--   repo: git@github.com:pre-commit/pre-commit-hooks
+-   repo: https://github.com/pre-commit/pre-commit-hooks.git
```

Comment:
these are for travis. otherwise,

```
[INFO] Initializing environment for git@github.com:pre-commit/pre-commit-hooks.
An unexpected error has occurred: CalledProcessError: Command: ['git', 'clone', '--no-checkout', 'git@github.com:pre-commit/pre-commit-hooks', '/home/travis/.pre-commit/repoA8NLi4']
Return code: 128
Expected return code: 0
Output: (none)
Errors: 
    Cloning into '/home/travis/.pre-commit/repoA8NLi4'...
    Warning: Permanently added the RSA host key for IP address '192.30.252.131' to the list of known hosts.
    Permission denied (publickey).
    fatal: Could not read from remote repository.
    
    Please make sure you have the correct access rights
    and the repository exists.
```